File: 1/1.9/Go/1.9.go
package main

import (
	"fmt"
	"strings"
)

func isSubstring(str string, substring string) bool {
	if strings.Contains(str, substring) {
		return true
	} else {
		return false
	}
}

func isCiclicShift(s1 string, s2 string) bool {
	if len(s1) != len(s2) {
		return false
	}
	return isSubstring(s2+s2, s1)
}
func main() {

	fmt.Print("waterbottle ", "erbottlewat ", isCiclicShift("waterbottle", "erbottlewat"), "\n")
	fmt.Print("Hello ", "oHell ", isCiclicShift("Hello", "oHell"), "\n")
	fmt.Print("Hello ", "Helloo ", isCiclicShift("Hello", "Helloo"), "\n")
	fmt.Print("Hello ", "Hoell ", isCiclicShift("Hello", "Hoell"), "\n")

}

File: 1/1.9/Python/1.9.py
def isSubstring(string: str, substring: str) -> bool:
    if substring in string:
        return True
    else:
        return False

def isCiclicShift(s1: str, s2: str) -> bool:
    if len(s1) != len(s2):
        return False
    return isSubstring(s2+s2, s1)

if __name__ == '__main__':
    print("waterbottle", "erbottlewat", isCiclicShift("waterbottle", "erbottlewat"))
    print("Hello", "oHell", isCiclicShift("Hello", "oHell"))
    print("Hello", "Helloo", isCiclicShift("Hello", "Helloo"))
    print("Hello", "Hoell", isCiclicShift("Hello", "Hoell"))

